From 212248a2b6b3a976e5be5060756f052f9fc836b7 Mon Sep 17 00:00:00 2001
From: jim <jim.yuan@supermicro.com>
Date: Tue, 5 Mar 2019 11:57:40 -0800
Subject: [PATCH] add imageId for P8 hostboot.

---
 src/build/tools/addimgid | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/build/tools/addimgid b/src/build/tools/addimgid
index d78df3c87..4cf7143f6 100755
--- a/src/build/tools/addimgid
+++ b/src/build/tools/addimgid
@@ -38,6 +38,10 @@ my $PREFIX = $ENV{'CROSS_PREFIX'};
 my $address = hex `${PREFIX}nm $src -C | grep $imageIdSym | colrm 17`;
 my $imageId = $ENV{'HOSTBOOT_VERSION'};
 if ($imageId eq '')
+{
+    $imageId = $ENV{'HOSTBOOT_P8_VERSION'};
+}
+if ($imageId eq '')
 {
     $imageId = `git describe --dirty || echo Unknown-Image \`git rev-parse --short HEAD\``;
 }
-- 
2.16.2.windows.1

