From d7b011cfea54562dc59cce70db8b12af33061b81 Mon Sep 17 00:00:00 2001
From: Chris Cain <cjcain@us.ibm.com>
Date: Sat, 5 Nov 2016 15:00:40 -0700
Subject: [PATCH] Open linear window for sram for channel 2

---
 src/occ/cmdh/cmdh_fsp.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/occ/cmdh/cmdh_fsp.c b/src/occ/cmdh/cmdh_fsp.c
index 2b79eb6..7c0d97f 100755
--- a/src/occ/cmdh/cmdh_fsp.c
+++ b/src/occ/cmdh/cmdh_fsp.c
@@ -416,7 +416,11 @@ errlHndl_t cmdh_fsp_init(void)
     // ----------------------------------------------------
     // Open Up Linear Window for FSP Communication
     // ----------------------------------------------------
-
+    TRAC_ERR("CJC: Writing linear window (ch2, base=0x%08X, log_size=0x%08X", CMDH_LINEAR_WINDOW_BASE_ADDRESS, LOG_SIZEOF_FSP_CMD_BUFFER);
+    ocb_linear_window_initialize(2, CMDH_LINEAR_WINDOW_BASE_ADDRESS, LOG_SIZEOF_FSP_CMD_BUFFER);
+    ocb_allow_untrusted_initialize(1, 1);
+    ocb_allow_untrusted_initialize(2, 1);
+    
     if(G_occ_interrupt_type == FSP_SUPPORTED_OCC)
     {
         // ----------------------------------------------------
-- 
2.9.2.windows.1

