From 9f5cc5f9c360152feaa5f069e9314e8b839eeb83 Mon Sep 17 00:00:00 2001
From: leoluo <leoluo@supermicro.com>
Date: Thu, 17 Mar 2016 14:25:24 -0700
Subject: [PATCH] change OPAL esel command to IBM 0x3a

Signed-off-by: Jim Yuan <jim.yuan@supermicro.com>
---
 include/ipmi.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/ipmi.h b/include/ipmi.h
index a6791e4..0d7e644 100644
--- a/include/ipmi.h
+++ b/include/ipmi.h
@@ -121,7 +121,7 @@
 #define IPMI_SET_SENSOR_READING		IPMI_CODE(IPMI_NETFN_SE, 0x30)
 
 /* AMI OEM comamnds. AMI uses NETFN 0x3a and 0x32 */
-#define IPMI_PARTIAL_ADD_ESEL		IPMI_CODE(0x32, 0xf0)
+#define IPMI_PARTIAL_ADD_ESEL		IPMI_CODE(0x3a, 0xf0)
 #define IPMI_PNOR_ACCESS_STATUS 	IPMI_CODE(0x3a, 0x07)
 
 /*
-- 
1.8.4.msysgit.0

