From 29e8d069880b688ad198a3ebaac17ec2f87a2127 Mon Sep 17 00:00:00 2001
From: jim <jim.yuan@supermicro.com>
Date: Tue, 11 Oct 2016 10:03:01 -0700
Subject: [PATCH] change pnor response command to 0x7

---
 src/include/usr/ipmi/ipmiif.H | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/usr/ipmi/ipmiif.H b/src/include/usr/ipmi/ipmiif.H
index e0b9668..aa26124 100644
--- a/src/include/usr/ipmi/ipmiif.H
+++ b/src/include/usr/ipmi/ipmiif.H
@@ -279,7 +279,7 @@ namespace IPMI
     { return std::make_pair(NETFUN_IBM, 0x07); }
 
     inline const command_t pnor_response(void)
-    { return std::make_pair(NETFUN_IBM, 0x08); }
+	{ return std::make_pair(NETFUN_IBM, 0x07); }
 
     // $TODO RTC:123256 - need to add code to get dcmi capabilities
     // This is a dcmi message used to request the
-- 
2.9.2.windows.1

